### ISSUE TITLE
Enchantment Suggestions?

### ISSUE THREAD

**Issue by tankyap** [TinyModularThings/UniqueEnchantments#74]
### 1. Sniper

Pros:
-   Increase damage depend on how far the projectile goes . `Increase extra damage by` `20% for every block the projectile travels horizontally (+25% of` `damage/block).`
- `Bows` & `Crossbows` can be enchanted with this enchantment
- The bullet drop would be a little bit lesser depend on the enchantment level.
   [X]Can penetrate at least `3 targets`

Cons:
- Enchantment level can only capped at II (Level 2) , `it would be extra 40% on this` `enchant level`
- Incompatible with `Power` enchantment.
- `x amount of % would be less than 3 quarters ( (1/4) *(x) )` when using fireworks on crossbows.

Ways to maximize the efficiency:
- `Stay as far as possible` between you and the target
- Works well with `Headshot` enchantment which is mentioned below
__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __

### 2. Headshot

Pros:
- You can now give an enemy a headshot by landing the projectile on their head! `The amount` `of headshot dmg for the projectile that is shot and landed on the head depends on the head shot multiplier(by%)`
- `Each number % of the multiplier is sorted by +100%, +125%, +150%` on each enchantment level (capped at level III)
- `Bow` & `Crossbow` can be enchanted with this enchantment

Cons:
- Fireworks that are shot by crossbows `cannot work with this enchantment`
- `The headshot multiplier would be deducted by small amount` depend on the helmet target wearing on.

Ways to maximize the efficiency:
-  Be precise and `know what you are doing`
- Works pretty well with `Sniper` enchantment which is mentioned above

**Comment by Xaikii** [comment]
Hey, thanks for the suggestions :) 

To give some direct feedback, 
1. doable and neat, however piercing is not something we'd like to do/add 
2. How exactly did you imagine us checking for Headshots? As far as I am aware, there is no way to check for that for every entity.

**Comment by tankyap** [comment]
> Hey, thanks for the suggestions :)
> 
> To give some direct feedback,
> 
>     1. doable and neat, however piercing is not something we'd like to do/add
> 
>     2. How exactly did you imagine us checking for Headshots? As far as I am aware, there is no way to check for that for every entity.

1.Yeah I think it just to op with piercing
2.I don't really know how just know,maybe add it to your list and add it anytime when you figure it out.My suggestion would be taking some examples from other mods maybe.

**Comment by tankyap** [comment]
Ayo, quick question here, just got a crash startup after I installed "Unique Enchantment Base" (1.19.2-3.02) mod and here are the crash reports and logs,cant find out the reason.
Click them below:

[crash report](https://github.com/TinyModularThings/UniqueEnchantments/files/10245870/crash-2022-12-16_20.41.28-fml.txt)

[latest.log file](https://github.com/TinyModularThings/UniqueEnchantments/files/10245877/latest.log)

**Comment by Xaikii** [comment]
It says you do not have the Unique Echantments Base installed.
https://www.curseforge.com/minecraft/mc-mods/unique-enchantments-base/files/4158104

**Comment by tankyap** [comment]
Yes I know but just because I installed the base version it crashes

**Comment by Xaikii** [comment]
then send me the log you get with the base mod added.
To say it upfront, Apotheosis is incompatible in 1.19.2 unless you're ready to leave some features out.
